### ISSUE TITLE
Command not found error after commit 80a89b9 

### ISSUE THREAD

**Issue by thepreacher** [marlonrichert/zsh-autocomplete#540]
Autocomplete stopped working after updating to commit 80a89b9. There's no type ahead showing and when I press tab, I get:
`
autocomplete:new:_main_complete:14: command not found: :autocomplete:old:_main_complete
`
## Environment
```zsh
apple darwin21.3.0 /usr/local/bin/zsh -zsh zsh-5.9-0-g73d3173

/Users/me/Git/zsh-snap/functions/.znap.source:26
/Users/me/Git/zsh-snap/functions/znap:42
/Users/me/.config/zsh/rc.d/04-plugins.zsh:120
/Users/me/.config/zsh/.zshrc:29
/Users/me/.config/zsh/.zshrc:16

80a89b9 (grafted, HEAD -> main, origin/main, origin/HEAD) Remove VS Code settings
```

* Operating system:  macOS
* Terminal emulator: Iterm 

## Steps to reproduce
```zsh
Cloning into 'zsh-autocomplete'...
remote: Enumerating objects: 57, done.
remote: Counting objects: 100% (57/57), done.
remote: Compressing objects: 100% (55/55), done.
remote: Total 57 (delta 0), reused 29 (delta 0), pack-reused 0
Receiving objects: 100% (57/57), 1.64 MiB | 9.17 MiB/s, done.
% brew autocomplete:new:_main_complete:14: command not found: :autocomplete:old:_main_complete
```

**Comment by haShinui** [comment]
Hello I am new here.
I am using manjora kde with the kde konsole and I am running oh my zsh on it.
 I removed the ":"  from` :autocomplete:old:_main_complete "$@"`. I am running oh my zsh and added `source ~/Git/zsh-autocomplete/zsh-autocomplete.plugin.zsh` at the beggining and added `zsh-autocomplete` to `plugins`. But I still get the `plugin 'zsh-autocomplete' not found` error.

**Comment by haShinui** [comment]
So I ran the ` rm $_comp_dumpfile` command and then reloaded my zsh session and autocomplete still doesn't work. I get the same message that it wasn't found. Any idea what it could be and how to fix it?

**Comment by marlonrichert** [comment]
Fixed: 5cc9da132e7535a540fb1235ce27fd5a233d4f0e
